backend/clubs/serializers.py

```diff
@@ -2975,6 +2975,37 @@ class Meta:
         )
 
 
+class ApprovalHistorySerializer(serializers.ModelSerializer):
+    approved = serializers.BooleanField()
+    approved_on = serializers.DateTimeField()
+    approved_by = serializers.SerializerMethodField("get_approved_by")
+    approved_comment = serializers.CharField()
+    history_date = serializers.DateTimeField()
+
+    def get_approved_by(self, obj):
+        user = self.context["request"].user
```

Comment:
Instead of casing on permissions here, it might be better use a permission class on the route (as Avi suggested) since only officer+ will be able to view this information.